scripts/cli/cli_helper.py

```diff
@@ -1,53 +1,42 @@
 import argparse
 import json
-import os
+import sys
 from datetime import datetime
 from os import environ
 from typing import List, NamedTuple, Optional
 
 from dateutil import parser, tz
 from linz_logger import get_log
 
-from scripts.files.files_helper import get_file_name_from_path
-
 
 class TileFiles(NamedTuple):
     output: str
     input: List[str]
 
 
-def format_source(source: List[str]) -> List[TileFiles]:
-    """Transform a list of file names (local) or dictionaries (Argo Workflows) to a list of `TileFiles`
-    When using locally `--source /path/to/BX24_500_031020.tif`, the file name must have the correct tilename.
+def format_source(source: str) -> List[TileFiles]:
+    """Transform a list of dictionaries (Argo Workflows) to a list of `TileFiles`
 
     Args:
-        source: a list of file names or containing a stringify list of dictionary
+        source: file contents string
 
     Returns:
         a list of `TileFiles` namedtuple
 
     Example:
     ```
-    >>> format_source(["[{'output': 'CE16_5000_1001', 'input': ['s3://bucket/SN9457_CE16_10k_0501.tif']}]"])
+    >>> format_source("[{'output': 'CE16_5000_1001', 'input': ['s3://bucket/SN9457_CE16_10k_0501.tif']}]")
     [TileFiles(output='CE16_5000_1001', input=['s3://bucket/SN9457_CE16_10k_0501.tif'])])]
-    >>> format_source(["s3://bucket/SN9457_CE16_10k_0501.tif", "s3://bucket/SN9457_CE16_10k_0502.tif"])
-    [TileFiles(output='output', input=['s3://bucket/SN9457_CE16_10k_0501.tif', 's3://bucket/SN9457_CE16_10k_0502.tif'])])]
     ```
     """
-    if source[0].startswith("[{"):
-        try:
-            source_json: List[TileFiles] = json.loads(
-                source[0], object_hook=lambda d: TileFiles(input=d["input"], output=d["output"])
-            )
-            return source_json
-        except json.JSONDecodeError as e:
-            get_log().debug("Decoding Json Failed", msg=e)
-
-    local_tile_file: List[TileFiles] = []
-    for s in source:
-        local_tile_file.append(TileFiles(output=os.path.splitext(get_file_name_from_path(s))[0], input=[s]))
-
-    return local_tile_file
+    try:
+        source_json: List[TileFiles] = json.loads(
+            source, object_hook=lambda d: TileFiles(input=d["input"], output=d["output"])
+        )
+    except json.JSONDecodeError as e:
+        get_log().error("Decoding Json Failed", msg=e)
+        sys.exit(1)
```

Comment:
I would prefer not terminate the program from this function. It would be a better approach to raise an exception and let the caller to handle it (and exit the program if needed). I think this has been done here https://github.com/linz/topo-imagery/pull/117